packages/presentation/lib/src/app/app_bloc.dart

```diff
@@ -140,7 +140,7 @@ class _AppBloc extends BlocImpl<BaseArguments, AppData> implements AppBloc {
   }
 
   @override
-  void bottomBarNavigation(int index) {
+  void navigationBar(int index) {
```

Comment:
naming

```suggestion
  void onNavigationBarClicked(int index) {
```

or smth like this